streampipes-model/src/main/java/org/apache/streampipes/model/datalake/ProviderType.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.apache.streampipes.model.datalake;
+
+public enum ProviderType {
+    FOLDER("local"),
+    S3("S3");
+
+    private String providerValue;
+
+    // Constructor to assign the value
+    ProviderType(String providerValue) {
+        this.providerValue = providerValue;
+    }
+
+    // Method to get the value of the provider
+    public String getProviderValue() {
```

Comment:
This is currently unused, can we remove `providerValue` and just use the enum keys?